tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -41,6 +41,10 @@ var (
 	createdInstanceCreds *types.Credentials
 	// Temp directory ansible created on the windows host
 	ansibleTempDir = ""
+	// kubernetes-node-windows-amd64.tar.gz SHA512
+	// Value from https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md#node-binaries-1
+	expectedSha = "a88e7a1c6f72ea6073dbb4ddfe2e7c8bd37c9a56d94a33823f531e303a9915e7a844ac5880097724e44dfa7f4a9659" +
```

Comment:
`expectedSha` of what? If you're downloading multiple items, it'll be hard in future. Also, you've to comment that this sha is something that needs to change whenever you're updating the binaries.

Reply:
This is the expectedSha of the node binary tar. This tar includes kubelet and kubectl - which we transfer on the Windows node as a part of WSU. AFAIK, we are only downloading kubernetes node binary from upstream, as a part of WSU. hybrid-overlay and worker.ign resides on our repo of within the cluster. I'm not sure what you mean by it being hard in the future.
For now I can rename this SHA variable to reflect it is kube-tar specific and call out when it needs to change within the comments

Reply:
Yes please rename the variable to reflect this